src/components/Application/ApplicationHeader.tsx

```diff
@@ -13,40 +13,31 @@ export interface AppHeaderProps {
 const ApplicationHeader: React.FC<AppHeaderProps> = ({ template, currentUser, ChildComponent }) => {
   const { code, name } = template
   return (
-    <>
-      <Button as={Link} to={`/applications?type=${code}`} style={inlineStyles} icon>
-        <Icon name="angle left" />
-        {`${name} ${strings.LABEL_APPLICATIONS}`}
-      </Button>
-      <Header textAlign="center" style={inlineStyles}>
+    <Container id="application-area">
+      <Button
```

Comment:
You might find this is easier as a `<Label>` rather than a `<Button>` (looks more like a label in the designs to me)

Reply:
Done: a58df28